coucharchive

```diff
@@ -255,6 +257,35 @@ class ReplicationControl(object):
         self._last_successes = multiprocessing.Manager().list()
         self._last_errors = multiprocessing.Manager().list()
 
+        self.has_run_queue_control = has_run_queue_control
+        if self.has_run_queue_control:
+            def run_queue_control():
```

Comment:
Suggestion for readability and easier indentation: make it a class function! :slightly_smiling_face: 
```python
class ReplicationControl:
    def _run_queue_control_thread(self):
```

Reply:
:+1: